lib/task.js

```diff
@@ -43,12 +43,22 @@ class Task extends Subject {
 		this.title = task.title;
 		this.skip = task.skip || defaultSkipFn;
 		this.task = task.task;
+		this.taskOptions = task.options;
+
 	}
 
 	get subtasks() {
 		return this._subtasks;
 	}
 
+	shouldSuspendUpdateRenderer() {
+		try {
```

Comment:
Why are you try/catching a boolean?

Reply:
I catching it, because `task.options` can be undefined. Without catching it, it throws TypeError, if you don|t supply any option.